Source/Scene/Model.js

```diff
@@ -1512,9 +1513,9 @@ define([
         }
 
         // Modify draw program to receive shadows
-        if (frameState.shadowsEnabled && model.receiveShadows) {
+        if (shadowsEnabled && model.receiveShadows) {
             drawVS = ShadowMapShader.createShadowReceiveVertexShader(drawVS);
-            drawFS = ShadowMapShader.createShadowReceiveFragmentShader(drawFS, context);
+            drawFS = ShadowMapShader.createShadowReceiveFragmentShader(drawFS, frameState, 'v_normal');
```

Comment:
Maybe put a TODO here that this is a bit hacky and does not use the glTF attribute semantics.